toolchain/sem_ir/formatter.cpp

```diff
@@ -1203,6 +1203,11 @@ class Formatter {
   auto FormatArg(InstId id) -> void { FormatInstName(id); }
 
   auto FormatArg(InstBlockId id) -> void {
+    if (!id.is_valid()) {
+      out_ << "invalid";
```

Comment:
Minor, but I think printing `id` here would result in `<invalid>`, which maybe is worth the `<>` bracketing / standardized print?

```suggestion
      out_ << id;
```

Reply:
Generally the formatter tries not to print out raw IDs, because they're not meaningful to a reader of the formatted IR, so I think using the ID type's printing machinery in this case would make us marginally less consistent. I also think the formatter should be owning the details of the IR format, even for invalid IR, and not delegating.